extensions/functions_string.yaml

```diff
@@ -163,3 +164,29 @@ scalar_functions:
           - value: "fixedchar<L1>"
           - value: "varchar<L2>"
         return: "BOOLEAN"
+  -
+    name: lower
+    description: Transforms the string to lower case characters
```

Comment:
Can we specify using what rules? I assume there are specific utf8 rules around this that we should reference?

Reply:
I'm going to punt on the string transforms to a separate PR and try to scope this one down a bit, so I've removed these changes for the moment.